src/net.cpp

```diff
@@ -589,6 +590,27 @@ bool CNode::ReceiveMsgBytes(const char *pch, unsigned int nBytes)
             if (strCommand != NetMsgType::PONG && strCommand != NetMsgType::PING && strCommand != NetMsgType::ADDR &&
                 strCommand != NetMsgType::VERSION && strCommand != NetMsgType::VERACK)
             {
+                if (strCommand == NetMsgType::GET_GRAPHENE)
+                    LOG(GRAPHENE, "ReceiveMsgBytes GET_GRAPHENE\n");
+                if (strCommand == NetMsgType::GRAPHENEBLOCK)
+                    LOG(GRAPHENE, "ReceiveMsgBytes GRAPHENEBLOCK\n");
+                if (strCommand == NetMsgType::GRAPHENETX)
+                    LOG(GRAPHENE, "ReceiveMsgBytes GRAPHENETX\n");
+                if (strCommand == NetMsgType::GET_GRAPHENETX)
+                    LOG(GRAPHENE, "ReceiveMsgBytes GET_GRAPHENETX\n");
```

Comment:
this is a little bit inefficient... better to use  "if else" statements so we don't have to potentially evaluate every if statement... or do just on if statement with ||  connecting ...and then just one generic LOG statement.

Reply:
Sounds good. I went with the "else if" option.